pkgs/by-name/rw/rwalk/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+  stdenv,
+  darwin,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rwalk";
+  version = "0.8.7";
+
+  src = fetchFromGitHub {
+    owner = "cestef";
+    repo = "rwalk";
+    rev = "v${version}";
```

Comment:
`tag = "v${version}";`?

Reply:
done

Reply:
idk why but the ci is showing error. I'll be working on that issue tomorrow if it's from my end.

Reply:
I think that now you have changed to a tag, the `{src.rev}` is no longer defined on line 49, and you'd need to use a tagged link to the release notes too

Reply:
Hell yeah, could you verify now?

Reply:
Ofborg runs again automatically once you've changed the PR. Should have the results within the day

Reply:
I think before it was failing on one of the GHA evals though, and that passed now, so it looks like it'll be fine

Reply:
Awesome 🎉